hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/recon/ReconConfigKeys.java

```diff
@@ -32,9 +32,11 @@ private ReconConfigKeys() {
 
   public static final String OZONE_RECON_DATANODE_ADDRESS_KEY =
       "ozone.recon.datanode.address";
+  public static final String OZONE_RECON_ADDRESS_KEY =
+      "ozone.recon.address";
   public static final String OZONE_RECON_DATANODE_BIND_HOST_KEY =
       "ozone.recon.datanode.bind.host";
   public static final String OZONE_RECON_DATANODE_BIND_HOST_DEFAULT =
       "0.0.0.0";
-  public static final int OZONE_RECON_DATANODE_PORT_DEFAULT = 9865;
+  public static final int OZONE_RECON_DATANODE_PORT_DEFAULT = 9891;
```

Comment:
This is already committed? Should we remove this change from here ?

Reply:
Yes, it can be removed